src/BenchmarkDotNet/Portability/CodeGen.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace BenchmarkDotNet.Portability
+{
+    public static class CodeGenHelper
+    {
+        // AggressiveOptimization is not available in netstandard2.0, so just use the value casted to enum.
+        public const MethodImplOptions AggressiveOptimizationOption = (MethodImplOptions) 512;
```

Comment:
is this value ignored by the older runtimes?

cc @AndyAyersMS who added the annotations with ifdefs

Reply:
Yes, I believe it will be ignored by older .NET Core and all .NET Framework versions.